.travis.yml

```diff
@@ -3,6 +3,7 @@ node_js:
   - "6"
   - "8"
   - "10"
+  - "node"
```

Comment:
IMHO we should make it explicit and use 12 here

Reply:
My intention here is to fail early if a new release is coming. `node` === `latest` which is always the released stable version on nodejs.org.

Reply:
I understand that, but then we'd have our CI failing for reasons unrelated to any of our changes. Changing the supported Node versions should be something explicit so that we can release it as a breaking change if needed.

Reply:
Ah good point. I have changed it to use `12`.